### PR TITLE
Fix: The metrics view API response is not super-set of resources response keys

### PR DESCRIPTION
**Pull request by Pearl1594**
## Description
The metrics API has few properties missing that are present in the corresponding resource. This addressed the following issue: https://github.com/apache/cloudstack/issues/3831

## Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

### CODE REVIEW

File: api/src/main/java/org/apache/cloudstack/api/response/VolumeResponse.java [hunk 1]
```diff
@@ -293,7 +293,7 @@ public String getObjectId() {
         return this.getId();
     }
 
-    public Boolean isDestroyed() {
+    public Boolean getDestroyed() {
```
Comment (DaanHoogland) [inline api/src/main/java/org/apache/cloudstack/api/response/VolumeResponse.java]: boolean getter should adhere to is<attr>() pattern

**Comment by DaanHoogland** [comment]
I do not care much either way but we agreed on the getter convention for booleans a while back. thoughts @rhtyd ?

**Comment by weizhouapache** [comment]
@DaanHoogland my suggestion is, we can use lombok.
https://projectlombok.org/

**Comment by DaanHoogland** [comment]
@weizhouapache looks cool. @Pearl1594 @rhtyd is there a specific reason to go back on our getter convention, or any other reason we shouldn't use lombok? (name of the neightbourhood I live in;) after the indonesian isleland \o/)

**Comment by DaanHoogland** [comment]
@Pearl1594 @rhtyd can you meddle in on @weizhouapache 's idea? (lombok would actually make this a get<>() as it is Boolean and not boolean)

**Comment by Pearl1594** [comment]
@DaanHoogland I've reverted the particular function to it's original naming convention. the getter function need not be modified for this case.

**Comment by rhtyd** [comment]
That's how BeanUtils::copyProperties and most IDEs works. The problem of booleans having `is` than `get` breaks the use case to copy bean/properties.

**Comment by rhtyd** [comment]
@DaanHoogland this is a special case for the metrics apis that extend over resource list apis. Without using getters instead of issers, it cannot be fixed. Turns out the previously referred agreement is merely our own preference but not a standard convention for IDEs, and libraries.

**Comment by rhtyd** [comment]
@blueorangutan package

**Comment by rhtyd** [comment]
@blueorangutan test

**Comment by rhtyd** [comment]
@blueorangutan package

**Comment by rhtyd** [comment]
@blueorangutan test

**Comment by rhtyd** [comment]
@blueorangutan test

**Pull request merged**